src/PKSim.Core/Model/PopulationSimulation.cs

```diff
@@ -330,5 +337,15 @@ private DataColumn columnsFor(IndividualResults results, string organ, string co
             Values =  column.Values
          };
       }
+
+      public DataColumn VenousBloodColumnForIndividual(int individualId, string compoundName)
```

Comment:
do they still need to be public?

Reply:
We are using that from PKAnalysesTask, or what do you mean? Do we need this to be a datacolumn at all? And can we work from the results directly?

Reply:
just wondering if we need tthis sutff outside of the class. But we do.  I wasn't sure